pkg/operator/manifests/storage/elasticsearch/templates/service.yaml

```diff
@@ -0,0 +1,40 @@
+# Licensed to Apache Software Foundation (ASF) under one or more contributor
```

Comment:
Since we use internal ES for the demo purpose, supporting ES7 is enough. Please remove ES6